src/api/File.js

```diff
@@ -93,16 +142,14 @@ class File extends Item {
         const template = getProp(entry, 'content.url_template');
         const token = await TokenService.getReadToken(getTypedFileId(item.id), this.options.token);
 
-        if (getProp(entry, 'status.state') !== 'success' || !extension || !template || !token) {
```

Comment:
Does this have any unintended consequences for other thumbnail usages?

Reply:
So far, I haven't found any. This function is only called within the `updateCollection` function in the ContentExplorer.